### PR TITLE
Improve code coverage for uiconv package

### PR DESCRIPTION
**Pull request by Ashmita152**
Signed-off-by: Ashmita Bohara <ashmita.bohara152@gmail.com>

## Which problem is this PR solving?
Increase code coverage for uiconv package

## Short description of the changes
Added coverage for two missing if conditions.

### CODE REVIEW

File: cmd/anonymizer/app/uiconv/reader.go [hunk 1]
```diff
@@ -39,7 +39,7 @@ type Reader struct {
 func NewReader(capturedFile string, logger *zap.Logger) (*Reader, error) {
 	wd, err := os.Getwd()
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("cannot get working directory: %w", err)
 	}
 	logger.Sugar().Infof("Current working dir is %s", wd)
```
Comment (yurishkuro) [inline cmd/anonymizer/app/uiconv/reader.go]: I think this is a debugging left-over, I don't see `wd` actually being used, if so it's best to delete L40-44

File: cmd/anonymizer/app/uiconv/reader_test.go [hunk 2]
```diff
@@ -99,3 +102,19 @@ func TestReader_TraceInvalidJson(t *testing.T) {
...
 }
+
+func TestReader_GetWDError(t *testing.T) {
+	inputFile := "fixtures/trace_non_existent.json"
+
+	wd, err := os.Getwd()
+	require.NoError(t, err)
+	err = os.Chmod(wd, 0000)
+	require.NoError(t, err)
+	defer os.Chmod(wd, 0755)
```
Comment (yurishkuro) [inline cmd/anonymizer/app/uiconv/reader_test.go]: This is clever, but it's conceivable that the tests might run on a read-only source dir where chmod won't work. If we can get rid of Getwd per my comment above, let's do that instead.

**Review by yurishkuro — COMMENTED** [review]
let's try to simplify instead

**Comment by Ashmita152** [comment]
Thank you @yurishkuro I was also wondering why we were doing that. I have updated the PR removing those.

**Pull request merged**